client/plots/singleCellPlot.js

```diff
@@ -46,16 +46,12 @@ class singleCellPlot {
 		const state = this.getState(appState)
 		const q = state.termdbConfig.queries
 		this.tableOnPlot = appState.nav?.header_mode == 'hidden'
-
-		//read files data
-		const controlsDiv = this.opts.holder
```

Comment:
Looking at the SingleCellPlot I noticed the controls are not properly placed after this changes. If possible next time add me as a reviewer.